hieradata/common.yaml

```diff
@@ -541,7 +541,50 @@ govuk_cdnlogs::service_port_map:
   assets: 6515
   bouncer: 6516
 
-govuk_ci::master::pipeline_jobs: *deployable_applications
+govuk_ci::master::pipeline_jobs:
+  <<: *deployable_applications
+  async_experiments: {}
+  cdn-acceptance-tests: {}
+  content-register: {}
+  deployment:
+    source: 'git'
+    repository: 'git@github.gds:gds/deployment.git'
```

Comment:
Could we do something clever with the repository name? It's almost always going to be `git@github.gds:gds/<repo name>.git`, so unless it's different we could probably make an assumption in the template?